clients/go/admin/pkce/handle_app_call_back.go

```diff
@@ -43,7 +43,8 @@ func getAuthServerCallbackHandler(c *oauth.Config, codeVerifier string, tokenCha
 		var opts []oauth2.AuthCodeOption
 		opts = append(opts, oauth2.SetAuthURLParam("code_verifier", codeVerifier))
 
-		token, err := c.Exchange(context.Background(), req.URL.Query().Get("code"), opts...)
+		ctx := context.WithValue(context.Background(), oauth2.HTTPClient, client)
+		token, err := c.Exchange(ctx, req.URL.Query().Get("code"), opts...)
 		if err != nil {
```

Comment:
If using a proxy, we need to use the http client created in `auth_interceptor.go` to retrieve the token. So far, this client wasn't used.